opm/grid/CpGrid.hpp

```diff
@@ -181,7 +178,13 @@ namespace Dune
         /// \brief The type of the collective communication.
 
     typedef Dune::MPIHelper::MPICommunicator MPICommunicator;
-    typedef Dune::CollectiveCommunication<MPICommunicator> CollectiveCommunication;
+#if DUNE_VERSION_NEWER(DUNE_GRID, 2, 7)
+    using Communication = Dune::Communication<MPICommunicator>;
```

Comment:
Could we get away with just the `Communication` alias, or are there too many places that depend on the name `CollectiveCommunication`?

Reply:
Indeed we can.

Reply:
Arrgh, back to zero. I was hesitating but tried to be nice... There is no reward for that 
```
/usr/include/dune/grid/common/defaultgridview.hh:223:64: error: no type named ‘CollectiveCommunication’ in ‘Dune::GridDefaultImplementation<3, 3, double, Dune::CpGridFamily>::Traits’ {aka ‘struct Dune::CpGridTraits’}
  223 |     typedef typename Grid :: Traits :: CollectiveCommunication CollectiveCommunication;
      |                                                                ^~~~~~~~~~~~~~~~~~~~~~~
```
Just a waste of time.

Reply:
Too bad!